README.md

```diff
@@ -145,6 +145,16 @@ Volcini](https://plus.google.com/109312219443477679717/posts). It is available
 on [this stackoverflow
 thread](http://stackoverflow.com/questions/17241071/writing-to-fifo-file-linux-monoc).
 
+### Java
+```java
+void echoToFile(String str) throws FileNotFoundException {
+    final String file = "/dev/pi-blaster";
+    try (PrintWriter out = new PrintWriter(new FileOutputStream(file), true)) {
+        out.println(str);
+    }
+}
+```
```

Comment:
Could you add a link to a full example, if you have one on GitHub?

Reply:
Unfortunately my project is not on GitHub((( This is simply a function that emulates "echo" from Bash and self-complete (doesn't use any global variable). I could add a complete example, but that would consume 20 more lines of the readme file practically showing how to concatenate strings in Java and invoke that function. I'm not sure we should grow the readme for this. What do you think?